### PR TITLE
Add gapic-google-cloud-logging-v2-docs

### PR DESCRIPTION
**Pull request by ethanbao**
Add gapic-google-cloud-logging-v2-docs

### CODE REVIEW

File: generated/python/gapic-google-cloud-logging-v2-docs/docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+=============================================================================================================
+
+This is the API documentation for ``gapic-google-cloud-logging-v2``.
+
+gapic-google-cloud-logging-v2 uses google-gax_ (Google API extensions) to provide an
+easy-to-use client library for the `Stackdriver Logging API`_ (v2) defined in the googleapis_ git repository
+
+
+.. _`google-gax`: https://github.com/googleapis/gax-python
+.. _`googleapis`: https://github.com/googleapis/googleapis/tree/master/google/google/cloud/logging/v2
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2-docs/docs/index.rst]: google/cloud/logging -> google/logging

File: generated/python/gapic-google-cloud-logging-v2-docs/docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+# Latex figure (float) alignment
+#'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+  (master_doc, 'gapic-google-cloud-logging-v2.tex', u'gax-google-cloud-logging-v2 Documentation',
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2-docs/docs/conf.py]: please do a global s/gax-google-/gapic-google- in this file

**Comment by geigerj** [comment]
Also, I accidentally submitted some comments on your old commit, which looks like it got overridden. Please disregard those.

**Comment by ethanbao** [comment]
PTAL

File: generated/python/gapic-google-cloud-logging-v2-docs/docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+# Latex figure (float) alignment
+#'figure_align': 'htbp',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+latex_documents = [
+  (master_doc, 'gapic-google-cloud-logging-v2.tex', u'gapic-googlecloud-logging-v2 Documentation',
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2-docs/docs/conf.py]: please fix (googlecloud -> google-cloud) here and throughout this file

File: generated/python/gapic-google-cloud-logging-v2-docs/README.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
+GAPIC library for the Stackdriver Logging API
+================================================================================
+
+gapic-google-cloud-logging-v2 uses google-gax_ (Google API extensions) to provide an
+easy-to-use client library for the `Stackdriver Logging API`_ (v2) defined in the googleapis_ git repository
+
+
+.. _`googleapis`: https://github.com/googleapis/googleapis/tree/master/google/cloud/logging/v2
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2-docs/README.rst]: please fix google/cloud/logging -> google/logging

File: generated/python/gapic-google-cloud-logging-v2-docs/docs/index.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+=============================================================================================================
+
+This is the API documentation for ``gapic-google-cloud-logging-v2``.
+
+gapic-google-cloud-logging-v2 uses google-gax_ (Google API extensions) to provide an
+easy-to-use client library for the `Stackdriver Logging API`_ (v2) defined in the googleapis_ git repository
+
+
+.. _`google-gax`: https://github.com/googleapis/gax-python
+.. _`googleapis`: https://github.com/googleapis/googleapis/tree/master/google/cloud/logging/v2
```
Comment (geigerj) [inline generated/python/gapic-google-cloud-logging-v2-docs/docs/index.rst]: please fix google/cloud/logging -> google/logging

**Comment by ethanbao** [comment]
Sorry, just did amend again.

**Pull request merged**